title_case/title_case_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rspec'
+require_relative 'title_case'
+
+describe 'title_case' do
```

Comment:
`nitpick` we could write `#title_case` since it refers to a method. [betterspecs](http://www.betterspecs.org/#describe)